digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/process/ProjectSectionCalculator.scala

```diff
@@ -194,8 +194,8 @@ object ProjectSectionCalculator {
             })
       }
     }
-    val groupedProjectLinks = newProjectLinks.groupBy(record => (record.roadNumber, record.roadPartNumber))
-    val groupedOldLinks = oldProjectLinks.groupBy(record => (record.roadNumber, record.roadPartNumber))
+    val groupedProjectLinks = newProjectLinks.sortBy(_.startAddrMValue).groupBy(record => (record.roadNumber, record.roadPartNumber))
```

Comment:
better to do mapValues(_.sortBy...) and not sort before grouping.

Reply:
I'll change this

Reply:
I think this is not necessary anymore